packages/graphql-html-formatter/package.json

```diff
@@ -11,10 +11,12 @@
     "node": ">=12.0"
   },
   "devDependencies": {
-    "@debugr/core": "^2.0"
+    "@debugr/apollo": "^3.0.0",
+    "@debugr/core": "^3.0"
   },
   "peerDependencies": {
-    "@debugr/core": "^2.0"
+    "@debugr/apollo": "^3.0.0",
```

Comment:
tady je ta závislost asi kvůli tý definici těch data shapes, co? ideálně by neměla bejt, můžeš mít i jinou graphql server implementaci než apollo a debugr by s ní mělo jít integrovat bez závislosti na apollu..

Reply:
navrhy?

Reply:
No... Mas pravdu, ze se mi to nelibi :D. Vzrostl by nam Debugr na 18 balicku. To se mi vic libi to dostat do formatteru...

Reply:
Ale zase spojovat dohromady formattery... To je taky trosku pakarna...